spec.md

```diff
@@ -173,7 +173,8 @@ graph LR
 A profile data model is a document with the following properties:
 
 * The document **MUST** have a `metadata` object with the following properties:
-    * The `id` property **MUST** be present and **MUST** be a DID. 
+    * The `id` property **MUST** be present. It must represent a unique indentifier within a governance framework.
+    * The `owner` property **MAY** be present. It **MUST** represent a DID of the owner of the document if present. 
```

Comment:
Question from ToIP @a-fox : Could it be a VID vs. a DID? 
Dan Bachenheimer : Important issuer binds to DID/VID.